modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -332,6 +332,14 @@ public Page<ObjectEntry> getObjectEntries(
 				accountEntries, AccountEntry::getAccountEntryId);
 		}
 
+		int start = QueryUtil.ALL_POS;
```

Comment:
Should this be ordered alphabetically?

Reply:
In the portal, I found similar cases where they are sorted alphabetically but I also found cases where they are sorted by logic (as I did), like in [here](https://github.com/liferay/liferay-portal/blob/dd3405ce5fe38c8989049dbd4274971e0ffdb0f1/modules/apps/archived/sync-service/src/main/java/com/liferay/sync/service/impl/SyncDLObjectServiceImpl.java#L707-L709). 

I also found this [thread](https://liferay.slack.com/archives/C1SJ64S9X/p1612519027001100) but I still don't now what's the right way.